solutions/quickstart/main.tf

```diff
@@ -0,0 +1,165 @@
+    "vpc-166" = {
+      destination = "166.8.0.0/14"
+      action      = "deliver"
+    }
+  }
+}
+
+module "existing_vpc_crn_parser" {
+  source  = "terraform-ibm-modules/common-utilities/ibm//modules/crn-parser"
+  version = "1.0.0"
+  crn     = var.existing_vpc_crn
+}
+
+resource "ibm_is_vpc_address_prefix" "client_to_site_address_prefixes_zone_1" {
+  name = var.prefix != null ? "${var.prefix}-client-to-site-address-prefixes-1" : "client-to-site-address-prefixes-1"
+  zone = local.zone_1
+  vpc  = local.existing_vpc_id
+  cidr = "10.10.40.0/24"
+}
+
+resource "ibm_is_network_acl" "client_to_site_vpn_acl" {
+  name = var.prefix != null ? "${var.prefix}-client-to-site-acl" : "client-to-site-acl"
+  vpc  = local.existing_vpc_id
+  rules {
+    name        = "outbound"
+    action      = "allow"
+    source      = "0.0.0.0/0"
+    destination = "0.0.0.0/0"
+    direction   = "outbound"
+    udp {
+      source_port_min = 443
+      source_port_max = 443
+    }
+  }
+  rules {
+    name        = "inbound"
+    action      = "allow"
+    source      = "0.0.0.0/0"
+    destination = "0.0.0.0/0"
+    direction   = "inbound"
+    udp {
+      port_min = 443
+      port_max = 443
+    }
+  }
+}
+
+resource "ibm_is_subnet" "client_to_site_subnet_zone_1" {
+  depends_on      = [ibm_is_vpc_address_prefix.client_to_site_address_prefixes_zone_1]
+  name            = var.prefix != null ? "${var.prefix}-client-to-site-subnet-1" : "client-to-site-subnet-1"
+  vpc             = local.existing_vpc_id
+  ipv4_cidr_block = "10.10.40.0/24"
+  zone            = local.zone_1
+  network_acl     = ibm_is_network_acl.client_to_site_vpn_acl.id
+}
+
+module "vpn" {
+  source                        = "../.."
+  depends_on                    = [time_sleep.wait_for_security_group]
+  server_cert_crn               = local.secrets_manager_cert_crn
+  vpn_gateway_name              = var.prefix != null ? "${var.prefix}-${var.name}" : var.name
+  resource_group_id             = module.resource_group.resource_group_id
+  subnet_ids                    = local.subnet_ids
+  create_policy                 = true
+  vpn_client_access_group_users = var.vpn_client_access_group_users
+  access_group_name             = var.prefix != null ? "${var.prefix}-client-to-site-vpn-access-group" : "client-to-site-vpn-access-group"
+  secrets_manager_id            = module.existing_sm_crn_parser.service_instance
+  vpn_server_routes             = local.vpn_server_routes
+}
+
+# workaround for https://github.com/terraform-ibm-modules/terraform-ibm-client-to-site-vpn/issues/45
+resource "time_sleep" "wait_for_security_group" {
+  depends_on       = [module.client_to_site_sg.ibm_is_security_group]
+  create_duration  = "10s"
+  destroy_duration = "60s"
+}
+
+module "client_to_site_sg" {
+  source                       = "terraform-ibm-modules/security-group/ibm"
+  version                      = "2.6.2"
+  add_ibm_cloud_internal_rules = true
```

Comment:
probably do not need to add all the internal rules if the other is an allow-all?

Reply:
deleted and added all outbound rules. as well